### ISSUE TITLE
[Accessibility] | Skip invite emails roles button for current user

### ISSUE THREAD

**Issue by thuanhuynhh** [pkp/pkp-lib#11630]
## Issue Description
The current mandatory invitation email policy also poses GDPR risks. Sending an unsolicited email without user consent may violate the core principles of GDPR. (I and another librarian from around the world don’t live in the EU; we don’t have a GDPR policy.) 

I think we need add button Skip invite emails button for 3.5.0 for effective internal governance and legal compliance. Key use cases include:

- Initial Setup & Data Migration: This process involves creating accounts for existing editorial and staff teams. Sending mass invitation emails to people who are already members of the organization is inefficient and unnecessary.
- Internal Role Changes: When a staff member is assigned an additional role (ex: from Production editor to Journal Editor). They are already part of the system, requiring them to “accept invitation” for an administrative change is a redundant step.
- Offline Consent: When a user has given written or verbal consent (e.g. at a conference). The admin needs a way to create an account for them without sending them repeated invitation emails.

The proposed solution is not an “edit” button, but a deliberate option: “Assign Role Directly (Skip Email) like you Decline Submission and Skip send email to author”.

This option empowers the Journal Manager to perform administrative tasks based on the legal basis of a “legitimate interest” in managing their team. This gives OJS more flexibility and is actually more GDPR compliant in essential administrative activities.

Best Regards,
Thuan Huynh

**Comment by Tribunal33** [comment]
@thuanhuynhh have these key use cases been resolved with the suggested ideas in this post? https://forum.pkp.sfu.ca/t/how-to-skip-invite-roles-for-current-user/96167/5.  I am not sure we want to add any further ways around the new invite to role feature. 

Here were the ideas presented in the post. 

> Bulk Edit / Initial Setup & Data Migration: This process involves creating accounts for existing editorial and staff teams. Sending mass invitation emails to people who are already members of the organization is inefficient and unnecessary.

**Solution:** You can use the users XML plugin to batch edit or import large groups of users

> Internal Role Changes: When a staff member is assigned an additional role (ex: from Production editor to Journal Editor). They are already part of the system, requiring them to “accept invitation” for an administrative change is a redundant step.

**Solution:** Admin tools > Hosted Journal > Settings Wizard > Users > Edit User, you can add roles to existing users without them accepting invitations.

> Offline Consent: When a user has given written or verbal consent (e.g. at a conference). The admin needs a way to create an account for them without sending them repeated invitation emails.

**Solution:** Admin tools > Hosted Journal Settings Wizard > User > Add User, you can add users like you did in 3.4.
